File: account.py
#!/user/bin/env python
import cgi
import cgitb
cgitb.enable()
htmlTop = """Content-Type:text/html\n\n
<!DOCTYPE html>
<html>
        <head>
                <metacharaset="utf-8"/>
                <title> My Account Confirmation </title>
        </head>
        <body>
	"""
htmlTail = """
        </body>
</html>"""
def main():
	formData = cgi.FieldStorage()
	print(htmlTop)
	print("<p> username: {} </p>".format(formData.getvalue("username")))
	print("<p> email: {} </p>".format(formData.getvalue("email")))
	print("<p> password: {} </p>".format(formData.getvalue("password")))
	print(htmlTail)

	
	
main()

<form action="account.py" method="POST">
		username: <br> <input name="username"> <br>
		e-mail: <br> <input name="email"/> <br>
		password: <br> <input name="password"/> <br>
		password again: <br> <input name="password2"/> <br>
		<input type="submit" value="Submit">
		</form>

File: account.js
var account_list = []
document.getElementById("accounts").innerHTML = account_list
function createAccount() {
	username = document.getElementById("username").value;
	email = documnet.getElementById("email").value;
	password = document.getElementById("password").value;
	password2 = document.getElementById("password2").value;
	var a;
	a = "[" + username + ", " + email + ", " + password + "]";
	account_list.push(a);
	document.getElementById("accounts").innerHTML = "works"
	}